src/components/Datatable/Table/Loading/LoadingOverlay.tsx

```diff
@@ -15,14 +19,22 @@ const overlayStyles = css`
 const LoadingOverlayContainer = styled.div`
   ${overlayStyles}
 
-  ${Spinner} {
-    display: inline-block;
+  ${FlexContainer} {
+    display: inline-flex;
     position: sticky;
     left: 50%;
     top: 50%;
     transform: translate(-50%, -50%);
     margin: 4rem;
     background: ${getColor('graphite5H')};
+    padding: 0 0.5rem;
+    height: 2.5rem;
+    border-radius: 4px;
+
+    > span {
```

Comment:
What about to create just a simple `LoadingText` component? That would make this style rule not depending on the HTML structure.